GeneratorInterface/RivetInterface/python/genParticles2HepMC_cfi.py

```diff
@@ -3,4 +3,6 @@
 genParticles2HepMC = cms.EDProducer("GenParticles2HepMCConverter",
     genParticles = cms.InputTag("genParticles"),
```

Comment:
Done, added commented line just below of line 4, also added a cfg file under the test directory.